src/lib/datetime.ts

```diff
@@ -0,0 +1,56 @@
+import moment = require('moment')
```

Comment:
Are you sure we should have this file? It appears to have been deleted in a previous commit